### PR TITLE
Parse new messages

### PR DESCRIPTION
**Pull request by pleonex**
This PR adds many new messages. As a summary:
* Parse logs related to thread settings and show a summary at the end
* Parse 37 new messages
* Add 2 new warnings
* Improve markdown format by moving two columns
* Fix some bugs introduced with the Logger refactoring

### CODE REVIEW

File: logparser/devices/markdownformatdevice.py [hunk 1]
```diff
@@ -128,6 +129,8 @@ def write_configurations(self, state):
             self.write_statistics_bandwidth(state)
         if 'statistics_packet' in state and not state['no_stats']:
             self.write_statistics_packets(state)
+        if 'threads' in state and not state['no_stats']:
```
Comment (iblancasa) [inline logparser/devices/markdownformatdevice.py:132]: If the argument is "no_", should be directly:
```python
if 'threads' in state and state['no_stats']:
```

Please, could you change this in ```_initialize_state```? Thank you

File: logparser/utils.py [hunk 2]
```diff
@@ -413,3 +433,13 @@ def parse_sn(seqnum, base=10):
     high_sn = int(seqnum[0], base)
     low_sn = int(seqnum[1], base)
     return (high_sn << 32) | (low_sn)
+
+
+def get_transport_name(class_id):
+    """Get the transport name from the class ID."""
+    TRANSPORTS = {1: "UDPv4", 2: "UDPv6", 3: "INTRA", 5: "UDPv6@510",
```
Comment (iblancasa) [inline logparser/utils.py]: Transport 2 is UDPv6 but previously was SHMEM. We should indicate this

**Review by iblancasa — CHANGES_REQUESTED** [review]

**Comment by iblancasa** [comment]
Good job :+1:

File: logparser/devices/markdownformatdevice.py (hunk 1 above)
Comment (pleonex) [reply logparser/devices/markdownformatdevice.py:132]: After discussing with @iblancasa we decided that the behavior is correct.

**Review by iblancasa — APPROVED** [review]
Great!

**Pull request merged**